README.md

```diff
@@ -154,6 +154,12 @@ to
 ```
 c.ServerApp.contents_manager_class = 'cs3api4lab.CS3APIsManager'
 ```
+### Disable default file browser
+Copy page_config.json from jupyter-config/page_config.json to:
```

Comment:
I would suggest `jupyter labextension disable` as an easier way to enable/disable extensions. Maybe could consider that to the docker image as well.

Reply:
Of course, this is an option too, I have updated a readme to add this information and added commands for dockerfile instead of config